<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenderScreen</class>
 <widget class="QMainWindow" name="RenderScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1321</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="canvas" native="true">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>109</y>
      <width>1000</width>
      <height>491</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="selectSHAPE" native="true">
    <property name="geometry">
     <rect>
      <x>159</x>
      <y>9</y>
      <width>1001</width>
      <height>91</height>
     </rect>
    </property>
    <widget class="QPushButton" name="draw">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Draw</string>
     </property>
    </widget>
    <widget class="QPushButton" name="newProject">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>New Project</string>
     </property>
    </widget>
    <widget class="QPushButton" name="save">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>20</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="quit">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>20</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Quit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="help">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>20</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Help?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sortbyAbutton">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>30</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Sort/Area</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sortbyIDbutton">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>60</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Sort/Area</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sortbyPbutton">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>0</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Sort/Perimeter</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1321</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(144, 166, 164)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Active Shapes</string>
     </property>
    </widget>
    <widget class="QComboBox" name="shapeSelection">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>103</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
   </widget>
   <zorder>frame</zorder>
   <zorder>canvas</zorder>
   <zorder>selectSHAPE</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1321</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
